contenttype/contenttype.go

```diff
@@ -32,18 +38,23 @@ func IsJSONContentType(contentType string) bool {
 	return isContentType(contentType, JSONContentType)
 }
 
+// IsGRPCContentType checks for content type.
```

Comment:
Please remove this -- gRPC is not a content type but a transport type. There is no reason for us to have this method.

Everything else is fine, but the gRPC content type additions should be remove. Thanks.